ui/desktop/src/utils/loadEnv.ts

```diff
@@ -23,9 +23,7 @@ export function loadZshEnv(isProduction: boolean = false): void {
       const matches = line.match(/^([^=]+)=(.*)$/);
       if (matches) {
         const [, key, value] = matches;
-        if (!process.env[key]) {
-          process.env[key] = value;
-        }
+        process.env[key] = value;
```

Comment:
this will ensure it sets it based on profile if it has one. It won't override the preconfig tho